share/completions/mkvextract.fish

```diff
@@ -7,17 +7,46 @@
 # Attachment ID 1: type 'application/x-truetype-font', size 53532 bytes, file name 'some_font.ttf'
 # Chapters: 7 entires
 
-function __fish_print_mkvextract_attachments
-    mkvmerge -i $argv | string match 'Attachment ID*' | string replace -r '.*?(\d+).*? type \'(.*?)\'.*?file name \'(.*?)\'' '$1:\t$3 ($2)'
+function __fish_mkvextract_find_matroska_in_args
+    for c in (commandline -opc)[2..-1]
```

Comment:
Of note is that this will fail if there is no second token (because of our annoying habit of failing invalid array indices).

This doesn't occur here because this is only ever called if there is already a subcommand, so there's no pressing need to change it.